### PR TITLE
Update messages.it.xlf

### PR DESCRIPTION
**Pull request by alfredonodo**
Hi,
I have updated the Italian translation, it was a longer task than I expected :)
I revised my work, but there might be some mistakes. So from today I will use Ghostfolio in Italian and correct any errors (for example in some cases I am not sure if it is a command/action or a description/label).
For economic and financial terms, I used [this thread](https://forum.wordreference.com/threads/stock-shares-securities-options-equity.182011/).

### CODE REVIEW

**Review by dtslvr — CHANGES_REQUESTED** [review]
Can you set every state from "new" to "translated" please?

It general it looks good, thanks for your changes 😎
When you really use it, you will probably find smaller details, but these are quick fixes then.

File: apps/client/src/locales/messages.it.xlf [hunk 1]
```diff
@@ -8956,7 +8956,7 @@
       </trans-unit>
       <trans-unit id="5827613432388799534" datatype="html">
         <source>open-source-alternative-to</source>
-        <target state="new">open-source-alternative-to</target>
+        <target state="new">alternativa-open-source-a</target>
```
[the remark this answers is not in the document]
Comment (alfredonodo) [reply apps/client/src/locales/messages.it.xlf:8959]: Not sure here. I used the English string.

**Comment by alfredonodo** [comment]
> Can you set every state from "new" to "translated" please?
> 
> It general it looks good, thanks for your changes 😎 When you really use it, you will probably find smaller details, but these are quick fixes then.

I changed new to translated and I used &Xgrave; for all accented letters.

**Review by dtslvr — APPROVED** [review]
> I changed new to translated and I used &Xgrave; for all accented letters.

Thank you @alfredonodo. Not sure if the encoding is really needed. It's just harder to read 😒

**Comment by alfredonodo** [comment]
> > I changed new to translated and I used &Xgrave; for all accented letters.
> 
> Thank you @alfredonodo. Not sure if the encoding is really needed. It's just harder to read 😒

If you want, I can remove them with Find and Replace.

File: apps/client/src/locales/messages.it.xlf (hunk 1 above)
[the remark this answers is not in the document]
Comment (dtslvr) [reply apps/client/src/locales/messages.it.xlf]: `alternativa-open-source-a` was the correct one! 😳

**Pull request merged**